.circleci/config.yml

```diff
@@ -222,6 +213,7 @@ jobs:
       - run:
           name: Run Cypress Tests
           command: yarn cy:test:circle
+          no_output_timeout: 20m
```

Comment:
Increased the timeout so that the longest running spec (usually finishes within 10-15 minutes even at its slowest) can at least finish and produce the video artifact.